audio.js

```diff
@@ -1,12 +1,225 @@
+//Display audio element id
 const display = document.getElementById("display");
 
-$(".playback").click(function (e) {
-    e.preventDefault();
+//Power ON and OFF
+let toggle = document.getElementById("power-switch-input");
+display.innerHTML = 'Power Off';
 
-    var clip = $(this).next('audio').get(0);
+$('.playback').attr('disabled', 'disabled'); //audio off
 
-    display.innerHTML = clip.id;
+// console.log(typeof toggle.value);
 
-    clip.currentTime = 0;
-    clip.play();
+toggle.addEventListener('input', () => {
+    if ((toggle.value == '0' && bank.value == '0') || (toggle.value == '0' && bank.value == '1')) {
+        toggle.value = '1';
```

Comment:
`<input>` elements of type `checkbox` have a `checked` attribute which you can use instead of `value`. You also don't need to reassign it in the JS, as it'll automatically switch between true and false as the user clicks on the element.

Reply:
I'm aware of the checked attribute and did not want to use a boolean; I purposefully used a integer value to denote "on" or "off".